get_image_predictions.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import pickle
+from model_prediction import *
+
```

Comment:
@sparadiso: probably a silly question but what is the best way to reference/import the modules once i have moved these files to workbench? setting PYTHONPATH? sys.path.append?